src/build/webpack.daemon.config.js

```diff
@@ -11,6 +11,7 @@ const build_dir = __dirname + "/../../build/" + browser;
 
 let entry;
 switch (process.env.BROWSER) {
+  case "edge":
```

Comment:
oh my, wasn't it nice to have this switch statement ready 😉

Reply:
Absolutely